spec/Component.coffee

```diff
@@ -1573,3 +1573,79 @@ describe 'Component', ->
         ip = new noflo.IP 'data', 'foo'
         ip.scope = 'eh'
         sin1.post ip
+
+    describe 'using streams', ->
+      it 'should not trigger without a full stream without getting the whole stream', (done) ->
+        setTimeout ->
+          done()
```

Comment:
This is not very nice, it's adding 1 sec to all our Travis builds. Maybe there's another way to test incomplete streams?